sceptre/config/reader.py

```diff
@@ -158,48 +158,40 @@ def construct_stacks(self):
         :rtype: set
         """
         stack_map = {}
+        command_stacks = set()
+        stack_group_configs = {}
 
-        root = self.context.full_command_path()
-        if path.isfile(root):
-            todo = {root}
-        else:
-            todo = set()
-            for directory_name, sub_directories, files in walk(root):
-                for filename in fnmatch.filter(files, '*.yaml'):
-                    if filename.startswith('config'):
-                        continue
+        root = self.context.full_config_path()
+        command_path = self.context.full_command_path()
 
-                    todo.add(path.join(directory_name, filename))
+        for directory_name, sub_directories, files in walk(root):
+            for filename in fnmatch.filter(files, '*.yaml'):
+                if filename.startswith('config.'):
```

Comment:
is this a erroneous `.` after `config?

Reply:
No, I figured I'd make sure it was `config.<ext>` rather than potentially accidentally catching `configblah.yaml`, which could be a valid stack name 😄 

It's not much better really, we probably should filter for `config.ya?ml` or something like that but I didn't want to go that far.

Reply:
👍